src/plugins/intel_gpu/src/plugin/ops/convolution.cpp

```diff
@@ -233,61 +233,48 @@ static void CreateGroupConvolutionBackpropDataOp(Program& p, const std::shared_p
+                                      std::int64_t deformableGroupsNum,
+                                      bool bilinearInterpolationPad = false) {
     auto inputs = p.GetInputPrimitiveIDs(op);
     std::string layerName = layer_type_name_ID(op);
-
-    auto params = GetConvolutionParameters(op->get_pads_begin(), op->get_dilations(), op->get_strides(), op->get_group());
     auto outDims = op->get_output_shape(0);
 
     std::vector<cldnn::primitive_id> weights = {inputs[2]};
-    if (params.groups > 1) {
-        auto convPrim = cldnn::convolution(layerName,
-                                           inputs[0],
-                                           inputs[1],
-                                           weights,
-                                           {},
-                                           params.groups,
-                                           op->get_deformable_group(),
-                                           params.stride,
-                                           params.padding,
-                                           params.dilation,
-                                           CldnnTensorFromIEDims(outDims),
-                                           op->get_friendly_name());
-
-        p.AddPrimitive(convPrim);
-        p.AddPrimitiveToProfiler(op);
-    } else {
+    // Remove weights from inputs
+    inputs.erase(inputs.begin() + 2);
+    if (params.groups == 1) {
         std::string defConvLayerNameInterp = layerName + "_interp";
         std::string defConvLayerNameConv = layerName;
         cldnn::tensor kernel;
         auto weights_shape = op->get_input_shape(2);
-        size_t sidx = 2 + (params.groups > 1 ? 1 : 0);
+        size_t nonSpatialDimsNum = 2;
         if (weights_shape.size() == 3) {
             kernel = cldnn::tensor(cldnn::batch(1),
                                    cldnn::feature(1),
-                                   cldnn::spatial(weights_shape[sidx + 2],
-                                                  weights_shape[sidx + 1],
-                                                  weights_shape[sidx + 0]));
+                                   cldnn::spatial(weights_shape[nonSpatialDimsNum + 2],
+                                                  weights_shape[nonSpatialDimsNum + 1],
+                                                  weights_shape[nonSpatialDimsNum + 0]));
         } else {
             kernel = cldnn::tensor(cldnn::batch(1),
-                                   cldnn::feature(1),
-                                   cldnn::spatial(weights_shape[sidx + 1],
-                                                  weights_shape[sidx + 0],
-                                                  1));
+                    cldnn::feature(1),
+                    cldnn::spatial(weights_shape[nonSpatialDimsNum + 1],
+                                   weights_shape[nonSpatialDimsNum + 0],
+                                   1));
```

Comment:
I suppose that code alignment in these 4 lines was correct before your changes related to variable renaming.

Reply:
done